.gitignore

```diff
@@ -2,6 +2,6 @@
 /.build
 /Packages
 /*.xcodeproj
-Package.pins
+Package.resolved
```

Comment:
`Package.resolved` was already added in https://github.com/apple/swift-nio/commit/cb62f14df344e9bdab0dd7d133e00910de5aa463 so probably this is unnecessary.